cmd/commands/runtime.go

```diff
@@ -292,6 +307,172 @@ func createRuntimeOnPlatform(ctx context.Context, opts *model.RuntimeInstallatio
 	return runtimeCreationResponse.NewAccessToken, nil
 }
 
+func ensureClusterRequirements(ctx context.Context, kubeFactory kube.Factory, namespace string) error {
+	requirementsValidationErrorMessage := "cluster does not meet minimum requirements"
+	var specificErrorMessages []string
+	
+	client, err := kubeFactory.KubernetesClientSet()
+	if err != nil {
+		return fmt.Errorf(fmt.Sprintf("Cannot create kubernetes clientset: %v ", err))
+	}
+
+	req := validationRequest{
+		rbac: []rbacValidation{
+			{
+				Resource:  "ServiceAccount",
```

Comment:
wondering if in this case I should change the hard-coded verb values and resource names to constants defined in the store

Reply:
no need